### PR TITLE
Proper defaultvalue for checkbox fields

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/48237

## Description
Added to demisto-sdk format command check proper defaultvalue for checkbox fields (type: 8).
The optional defaultvalue values are: 'true', 'false'.

## Must have
- [x] Tests
- [x] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 * Added HelloWorldSlim template option for *--template* flag in **demisto-sdk init** command.
 * Fixed an issue where the HelloWorldSlim template in **demisto-sdk init** command had an integration id that was conflicting with HelloWorld integration id.
 * Updated the SDK to use demisto-py 3.1.6, allowing use of a proxy with an environment variable.
+* The **format** command will now make sure that checkbox parameters default value are 'true' or 'false'.
```
Comment (dorschw) [inline CHANGELOG.md:21]:
```suggestion
* The **format** command will now make sure that checkbox parameters default value are string 'true' or 'false'.
```

File: demisto_sdk/commands/format/tests/test_formatting_yml_test.py [hunk 2]
```diff
@@ -717,6 +717,27 @@ def test_format_beta_integration(self):
...
+        Then
+            - Check the field has changed to 'false'
+            - Check the field has changed to 'true'
+            - Check the field has changed to 'false'
+
+        """
+
+        formatter = IntegrationYMLFormat(input=SOURCE_FORMAT_INTEGRATION_DEFAULT_VALUE)
+        formatter.set_default_value_for_checkbox()
+        assert 'false' in formatter.data['configuration'][1]['defaultvalue']
```
Comment (dorschw) [inline demisto_sdk/commands/format/tests/test_formatting_yml_test.py:737]: `in`? isn't it `==` ?

File: demisto_sdk/commands/format/update_integration.py [hunk 3]
```diff
@@ -192,6 +192,16 @@ def update_beta_integration(self):
...
 
+    def set_default_value_for_checkbox(self):
+        """Check the boolean default value are true or false lowercase"""
+        config = self.data.get('configuration', {})
+        for param in config:
+            if param.get('type') == ParameterType.BOOLEAN.value:
+                if 'true' == str(param.get('defaultvalue')).lower():
+                    param['defaultvalue'] = 'true'
+                if 'false' == str(param.get('defaultvalue')).lower():
+                    param['defaultvalue'] = 'false'
```
Comment (dorschw) [inline demisto_sdk/commands/format/update_integration.py:203]:
```suggestion
            if param.get('type') == ParameterType.BOOLEAN.value \
                    and (value := param.get('defaultvalue') not in ('true', 'false')):
                if 'true' == str(value).lower():
                    param['defaultvalue'] = 'true'
                elif 'false' == str(value).lower():
                    param['defaultvalue'] = 'false'
```

**Review by dorschw — APPROVED** [review]
💯

File: demisto_sdk/commands/format/tests/test_formatting_yml_test.py (hunk 2 above)
Comment (barchen1) [reply demisto_sdk/commands/format/tests/test_formatting_yml_test.py]: yes, changed

File: CHANGELOG.md (hunk 1 above)
Comment (barchen1) [reply CHANGELOG.md]: done

File: demisto_sdk/commands/format/update_integration.py (hunk 3 above)
Comment (barchen1) [reply demisto_sdk/commands/format/update_integration.py]: done

**Pull request merged**